gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -470,6 +535,309 @@ def prepare_ht_for_validation(
     return ht
 
 
+def populate_subset_info_dict(
+    subset: str,
+    description_text: str,
+    pops: Dict[str, str] = POPS,
+    faf_pops: Dict[str, str] = FAF_POPS,
+    sexes: List[str] = SEXES,
+    label_delimiter: str = "_",
+) -> Dict[str, Dict[str, str]]:
+    """
+    Call `make_info_dict` to populate INFO dictionary for the requested `subset`.
+
+    Creates:
+        - INFO fields for AC, AN, AF, nhomalt for each combination of sample population, sex both for adj and raw data
+        - INFO fields for filtering allele frequency (faf) annotations
+
+    :param subset: Sample subset in dataset. "" is used as a placeholder for the full dataset.
+    :param description_text: Text describing the sample subset that should be added to the INFO description.
+    :param pops: Dict of sample global population names for gnomAD genomes. Default is POPS.
```

Comment:
maybe not for variable names, but as least for descriptions should we change "population" to "genetic ancestry group"?